### PR TITLE
Bring core privacy features forward

### PR DESCRIPTION
**Pull request by raksooo**
This PR brings core privacy features forward. The changes include:
- Move Multihop and DAITA to be accessible from `Settings.tsx`
- Move Multihop setting into a dedicated view
- Add illustrations and more information to the Multihop and DAITA views
- Remove multihop confirmation dialog

To be able to show the information in a paged layout in `DaitaSettings.tsx` I implemented the component `PageSlider.tsx`.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Line wrap the file at 100 chars.                                              Th
   also be used automatically when connecting fails with other methods.
 - Add feature indicators to the main view along with redesigning the connection details.
 - Add "Smart Routing" feature which simplifies connecting to DAITA-enabled relays.
+- Move DAITA and Multihop to the root settings view along with moving multihop into a dedicated
```
Comment (MrChocolatine) [inline CHANGELOG.md:30]: For consistency?

```suggestion
- Move DAITA and multihop to the root settings view along with moving multihop into a dedicated
```

Comment (raksooo) [reply CHANGELOG.md]: Fixed! Thanks!

**Review by hulthe — COMMENTED** [review]
> Reviewed 1 of 5 files at r1, 3 of 4 files at r4, 1 of 2 files at r5, 2 of 3 files at r7, 3 of 3 files at r8, 1 of 1 files at r9, 1 of 1 files at r10, 1 of 1 files at r11, 2 of 2 files at r12, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine and @raksooo)

___
*[`gui/src/renderer/components/PageSlider.tsx` line 159 at r11](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866#-O7j0ueh96XFPxrV2DYk:-O7j0ueh96XFPxrV2DYl:bxwn4mv) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/345a3450c96d60b4ebc05f4f8e55593b580b992a/gui/src/renderer/components/PageSlider.tsx#L159)):*
> ```XML
> 
>   [`${StyledTransparentButton}:hover &&`]: {
>     backgroundColor: props.$current ? colors.white80 : colors.white80,
> ```

white80 or white80?

**Review by raksooo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)* status: 4 of 13 files reviewed, 1 unresolved discussion (waiting on @hulthe and @MrChocolatine)

___
*[`gui/src/renderer/components/PageSlider.tsx` line 159 at r11](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866#-O7j0ueh96XFPxrV2DYk:-O7j4eZA0t-wawNtpSI2:b-p4yc7) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/345a3450c96d60b4ebc05f4f8e55593b580b992a/gui/src/renderer/components/PageSlider.tsx#L159)):*
<details><summary><i>Previously, hulthe (Joakim Hulthe) wrote…</i></summary><blockquote>

white80 or white80?
</blockquote></details>

Oops, fixed!

**Review by hulthe — APPROVED** [review]
> Reviewed 9 of 9 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)* status: all files reviewed, 1 unresolved discussion (waiting on @MrChocolatine)

**Review by MarkusPettersson98 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 5 files at r1, 3 of 4 files at r4, 9 of 9 files at r13, 9 of 9 files at r14, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866#-:-O80iCL50C3xg3IsA9ak:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine and @raksooo)

___
*[`gui/src/renderer/components/DaitaSettings.tsx` line 2 at r14](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866#-O80cqp23P1jfMp1iK2G:-O80cqp23P1jfMp1iK2H:bqh60jt) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/e1fe4582671a3d860153c16d050202b6643cfb44/gui/src/renderer/components/DaitaSettings.tsx#L2)):*
> ```XML
> import { useCallback } from 'react';
> import React from 'react';
> ```

Nit These imports could be consolidated into one line
```javascript
import React, { useCallback } from 'react';
```



_Code quote:_
```XML
import { useCallback } from 'react';
import React from 'react';
```

**Review by raksooo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)* status: 1 of 13 files reviewed, 1 unresolved discussion (waiting on @hulthe, @MarkusPettersson98, and @MrChocolatine)

___
*[`gui/src/renderer/components/DaitaSettings.tsx` line 2 at r14](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866#-O80cqp23P1jfMp1iK2G:-O80qY_k8fC8CgKTSbBQ:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/e1fe4582671a3d860153c16d050202b6643cfb44/gui/src/renderer/components/DaitaSettings.tsx#L2)):*
<details><summary><i>Previously, MarkusPettersson98 (Markus Pettersson) wrote…</i></summary><blockquote>

Nit These imports could be consolidated into one line
```javascript
import React, { useCallback } from 'react';
```
</blockquote></details>

Done.

**Review by hulthe — APPROVED** [review]
> Reviewed 3 of 4 files at r17, 1 of 2 files at r18, 2 of 3 files at r20, 3 of 3 files at r21, 1 of 1 files at r22, 1 of 1 files at r23, 1 of 1 files at r24, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6866)* status: all files reviewed, 1 unresolved discussion (waiting on @MrChocolatine)

**Pull request merged**